sources/packages/api/src/utilities/auth-utils.ts

```diff
@@ -54,14 +54,16 @@ export function setGlobalPipes(app: INestApplication) {
 }
 
 /**
- * Gets user full name considering that the first name
- * name can be null for mononymous names.
+ * Gets user full name considering that the user
+ * name can be a mononymous names.
  * @param user object with firstName and lastName.
  * @returns user full name.
  */
 export function getUserFullName(user: {
   firstName: string;
   lastName: string;
 }): string {
-  return `${user.firstName} ${user.lastName}`.trim();
+  return `${(user.firstName ?? "").trim()} ${(
+    user.lastName ?? ""
+  ).trim()}`.trim();
```

Comment:
trim() is used for $firstName and $lastName already, so is it again required at the end?

Reply:
Yes, because there is a white space in the middle that should be removed in case firstName or lastName are null or empty.

Reply:
Okay. Makes sense. Thanks